src/Categories/Diagram/Equalizer.agda

```diff
@@ -47,18 +47,17 @@ record IsEqualizer {E} (arr : E ⇒ A) (f g : A ⇒ B) : Set (o ⊔ ℓ ⊔ e) w
     equalize eq′ ≈˘⟨ eqk ⟩
     k            ∎
 
-  equality-∘ : f ∘ arr ∘ h ≈ g ∘ arr ∘ h
```

Comment:
This proof was removed as it's `extendʳ equality`, and doesn't really buy us that much.